testsuite/run_sets/finishing.yml

```diff
@@ -14,5 +14,6 @@
 # these features are needed for gathering log/debug info
 - features/finishing/srv_debug.feature
 - features/finishing/allcli_debug.feature
+- features/finishing/srv_rename_hostname.feature
```

Comment:
It was speculated that if the feature failed, it could cause a cascade of failures
However now the feature has been adapted to have a proper cleanup scenario so it can be moved to secondary, good point